### ISSUE TITLE
does it support round join?

### ISSUE THREAD

**Issue by mysisi** [spite/THREE.MeshLine#53]
Hi, your code inspired me a lot, but i still wonder how to support round join in meshline?
I have seen a demo https://www.shadertoy.com/view/lts3Df doing this work in fragmentShader, but it requires each point be a uniforms, look forward to your solution.

**Comment by spite** [comment]
No, MeshLine doesn't support round join. It could, by adding more geometry where necessary, but it's a bit outside of the scope of the library. Also, note that the shadertoy you linked is a spline rendering, not a round joint. You can create the same spline in MeshLine with any regular function that creates the spline points.
